jhub_apps/service/japps_routes.py

```diff
@@ -14,4 +16,8 @@
 @router.get("/edit-app", response_class=HTMLResponse)
 @router.get("/server-types", response_class=HTMLResponse)
 async def handle_apps(request: Request):
-    return templates.TemplateResponse("japps_home.html", {"request": request})
+    now = datetime.now()
```

Comment:
I think its worth passing in `timezone.utc` for consistency and reproducibility.

```python
from datetime import timezone
datetime.now(timezone.utc)
```

Reply:
Fixed